### ISSUE TITLE
Some suggestion for app improvement

### ISSUE THREAD

**Issue by flawsom** [windstone-aristotle-yellow/animeth#3]
how about letting the users chose their preferable languange while accessing the app
something as a translator ( u can use the apis put there ) it would just make it more accessible to the people

**Comment by notwindstone** [comment]
Yeah, seems like a good idea to me. But rather than using translation APIs, it would be better to manage translations with tools like crowdin

**Comment by flawsom** [comment]
yeah crowdin would be great to

**Comment by CrystalKnight1** [comment]
dscrd: smartpowerstage
